src/debonair/Bill.java

```diff
@@ -0,0 +1,62 @@
+//user defined package
+package debonair;
```

Comment:
specified package doesn't show a relationship with other packages of the same program modify to something like 
```suggestion
package com.debonair.billing;
```